docs/Contributing/API-for-contributors.md

```diff
@@ -1283,6 +1283,8 @@ If the `name` is not already associated with an existing team, this API route cr
 | mdm.macos_settings.custom_settings        | list   | body  | The list of .mobileconfig files to apply to hosts that belong to this team.                                                                                                                                                         |
 | scripts                                   | list   | body  | A list of script files to add to this team so they can be executed at a later time.                                                                                                                                                 |
 | mdm.macos_settings.enable_disk_encryption | bool   | body  | Whether disk encryption should be enabled for hosts that belong to this team.                                                                                                                                                       |
+| mdm.windows_settings                      | object | body  | The Windows-specific MDM settings.                                                                                                                                                                                                  |
+| mdm.windows_settings.custom_settings      | list   | body  | The list of .xml files to apply to hosts that belong to this team.                                                                                                                                                                  |
```

Comment:
Just for my understanding - those docs are in your PR because this is a contributor API endpoint, right? Otherwise it would've been addressed by Marko's PR?